app/src/main/res/values/styles.xml

```diff
@@ -103,4 +103,8 @@
         <item name="android:textSize">@dimen/bottom_sheet_text_size</item>
     </style>
 
+    <style name="full_screen_dialog" parent="android:Theme.Black.NoTitleBar.Fullscreen">
+        <item name="android:windowAnimationStyle">@android:style/Animation.Dialog</item>
+    </style>
+
```

Comment:
There is multiple conventions of naming styles in this file, we should standardize this.